lib/components/viewers/stop-time-cell.js

```diff
@@ -32,8 +32,8 @@ const StopTimeCell = ({
   // 48:00 hours (or more) from the start of the service day.
   const departureTimeRemainder = departureTime % ONE_DAY_IN_SECONDS
   const daysAfterServiceDay = (departureTime - departureTimeRemainder) / ONE_DAY_IN_SECONDS
-  const departureDay = serviceDay.add(daysAfterServiceDay, 'day')
-  const vehicleDepartsToday = now.dayOfYear() === departureDay.dayOfYear()
+  const departureDay = serviceDay?.add(daysAfterServiceDay, 'day')
+  const vehicleDepartsToday = now?.dayOfYear() === departureDay?.dayOfYear()
```

Comment:
I'm kinda curious why this change was made.

Reply:
I'm not sure. I think this was from Rob Gregg. I'll investigate and remove if not needed.

Reply:
`homeTimezone` was not being passed correctly to this component. This has been fixed and the optional chaining removed.